src/Query/DateParser.ts

```diff
@@ -1,16 +1,15 @@
 import * as chrono from 'chrono-node';
 import moment from 'moment';
+import { DateRange } from './DateRange';
 
 export class DateParser {
     public static parseDate(input: string, forwardDate: boolean = false): moment.Moment {
         // Using start of day to correctly match on comparison with other dates (like equality).
-        return window
-            .moment(
-                chrono.parseDate(input, undefined, {
-                    forwardDate: forwardDate,
-                }),
-            )
-            .startOf('day');
+        return moment(
+            chrono.parseDate(input, undefined, {
```

Comment:
I checked about this. Please revert the changes from window.moment() to moment().
See discussion on Discord starting 

https://discord.com/channels/686053708261228577/840286264964022302/1094296498939900024

I will write up an issue to investigate more thoroughly, but I would prefer to avoid the risk of changing behaviour in the release environment for now...

Reply:
Written up in #1849

Reply:
> Please revert the changes from window.moment() to moment(). See discussion on Discord starting
> 
> https://discord.com/channels/686053708261228577/840286264964022302/1094296498939900024


I realised that this is not urgent to do, as there were pre-existing uses of moment() in the previous release.

So don't worry about this, and I will go ahead and merge this now.